hpx/runtime/threads/thread_enums.hpp

```diff
@@ -49,8 +49,13 @@ namespace hpx { namespace threads
                                  without high priority rescheduling */
     };
 
+    ///////////////////////////////////////////////////////////////////////////
+    /// \brief Returns the thread state
+    ///
```

Comment:
This returns the *name* of the given state. Could you change the description to say that?